File: pset6/caesar.py
import cs50
import sys

if len(sys.argv) != 2:
    print("Usage: ./caesar k")
    exit(1)
else:
    k = sys.argv[1]
    plain = ""
    print("plaintext: ", end="")
    plain = cs50.get_string()
    print("ciphertext: ",end="")
    n = len(plain)
    for i in range(n):
        if plain[i].islower():
            print("{}".format(chr(((((ord(plain[i]) + int(k)) - 97) % 26) + 97))),end="")
        elif plain[i].isupper():
            print("{}".format(chr(((((ord(plain[i]) + int(k)) - 65) % 26) + 65))),end="")
        else:
            print("{}".format(plain[i]),end="")
    print("")
            

File: pset2/caesar.c
#include <cs50.h>
#include <ctype.h>
#include <stdio.h>
#include <string.h>

int main(int argc, string argv[])
{
   if(argc != 2){
       printf("Usage: ./caesar k");
       return 1;
   }
   else {
   int k = atoi(argv[1]);
   string plain;
   printf("%s","plaintext: ");
   plain = get_string();
   printf("%s","ciphertext: ");

   for (int i = 0, n = strlen(plain); i < n; i++)
    {

        if islower(plain[i])
        printf("%c", (((plain[i] + k) - 97) % 26) + 97);
        else if isupper(plain[i])
        printf("%c", (((plain[i] + k) - 65) % 26) + 65);
        else
        printf("%c", plain[i]);
    
   }
   printf("\n");
  }
   

   
}